### PR TITLE
OSSMDOC-214 Release Notes for Service Mesh 2.1

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Aligned team: Dev Tools
- For branches: 
- Jira: https://issues.redhat.com/browse/OSSMDOC-214
- Direct link to doc preview: https://deploy-preview-36858--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes.html#new-features-red-hat-openshift-service-mesh-2-1
- SME review: @rcernich 
- QE review: @yxun
- Peer review: @rolfedh 
- Cherry pick to 4.6, 4.7, 4.8, 4.9, 4.10

### CODE REVIEW

File: modules/ossm-rn-fixed-issues.adoc [hunk 1]
```diff
@@ -41,18 +41,22 @@ The following issues been resolved in the current release:
 
 * link:https://issues.redhat.com/browse/MAISTRA-2401[MAISTRA-2401] CVE-2021-3586 servicemesh-operator: NetworkPolicy resources incorrectly specified ports for ingress resources. The NetworkPolicy resources installed for {Productname} did not properly specify which ports could be accessed. This allowed access to all ports on these resources from any pod. Network policies applied to the following resources are affected:
```
Comment (rolfedh) [inline modules/ossm-rn-fixed-issues.adoc:43]:
```suggestion
+
```
Adding a plus to this line indents the following list relative to the preceding bulleted release note.

Comment (rolfedh) [reply modules/ossm-rn-fixed-issues.adoc:43]: Then, lines 44 through 50 need only one `*` per item.

Comment (rolfedh) [reply modules/ossm-rn-fixed-issues.adoc:43]: https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#ex-complex

File: modules/ossm-rn-fixed-issues.adoc [hunk 2]
```diff
@@ -41,18 +41,22 @@ The following issues been resolved in the current release:
...
+** Sidecar injector
 
 * link:https://issues.redhat.com/browse/MAISTRA-2378[MAISTRA-2378] When the cluster is configured to use OpenShift SDN with `ovs-multitenant` and the mesh contains a large number of namespaces (200+), the {product-title} networking plugin is unable to configure the namespaces quickly. {ProductShortName} times out causing namespaces to be continuously dropped from the service mesh and then reenlisted.
 
 * link:https://issues.redhat.com/browse/MAISTRA-2370[MAISTRA-2370] Handle tombstones in listerInformer. The updated cache codebase was not handling tombstones when translating the events from the namespace caches to the aggregated cache, leading to a panic in the go routine.
 
+* link:https://issues.redhat.com/browse/MAISTRA-2117[MAISTRA-2117] Add optional ConfigMap mount to operator. The CSV now contains an optional ConfigMap volume mount, which mounts the smcp-templates ConfigMap if it exists. If the smcp-templates ConfigMap does not exist, the mounted directory is empty. When you create the ConfigMap, the directory is populated with the entries from the ConfigMap and can be referenced in SMCP.spec.profiles. No restart of the operator is required.
```
Comment (rolfedh) [inline modules/ossm-rn-fixed-issues.adoc:56]:
```suggestion
* link:https://issues.redhat.com/browse/MAISTRA-2117[MAISTRA-2117] Add optional `ConfigMap` mount to operator. The CSV now contains an optional `ConfigMap` volume mount, which mounts the `smcp-templates` `ConfigMap`, if it exists. If the `smcp-templates` `ConfigMap` does not exist, the mounted directory is empty. When you create the `ConfigMap`, the directory is populated with the entries from the `ConfigMap` and can be referenced in `SMCP.spec.profiles`. No restart of the operator is required.
```

File: modules/ossm-rn-fixed-issues.adoc [hunk 3]
```diff
@@ -41,18 +41,22 @@ The following issues been resolved in the current release:
...
++
+Customers using the 2.0 operator with a modified CSV to mount the smcp-templates ConfigMap can upgrade to {ProductName} 2.1. After upgrading, you can continue using an existing ConfigMap, and the profiles it contains, without editing the CSV. Customers that previously used ConfigMap with a different name will either have to rename the ConfigMap or update the CSV after upgrading.
```
Comment (rolfedh) [inline modules/ossm-rn-fixed-issues.adoc:58]:
```suggestion
Customers using the 2.0 operator with a modified CSV to mount the `smcp-templates` `ConfigMap` can upgrade to {ProductName} 2.1. After upgrading, you can continue using an existing `ConfigMap`, and the profiles it contains, without editing the CSV. Customers that previously used `ConfigMap` with a different name must rename the `ConfigMap` or update the CSV after upgrading.
```

**Review by rolfedh — APPROVED** [review]
Left some suggestions. Pre-approving.

**Review by yxun — APPROVED** [review]

**Review by mattmahoneyrh — COMMENTED** [review]
In Known Issues section, the following Issues are now closed and can be removed from this known issues list:
 OSSM-419
 OSSM-291
 OSSM-289
 OSSM-287

File: modules/ossm-rn-known-issues.adoc [hunk 4]
```diff
@@ -23,7 +23,7 @@ These limitations exist in {ProductName}:
 
 * The Bookinfo sample application cannot be installed on IBM Z and IBM Power Systems.
 
-* WebAssembly is unsupported on IBM Z and IBM Power Systems.
+* WebAssembly is unsupported on IBM Z Systems.
```
Comment (SNiemann15) [inline modules/ossm-rn-known-issues.adoc:26]: @rh-tokeefe Can you please also remove "Systems".  
**IBM Z** is a brand name and **IBM Power Systems** is as well.

**Comment by rh-tokeefe** [comment]
Yes, thank you, and done @SNiemann15   =)

**Pull request merged**